src/index.js

```diff
@@ -0,0 +1,155 @@
+  // dueDate.textContent = todo.duedate;
+  // proprity.textContent = todo.priority;
+
+  panel.appendChild(description);
+  panel.appendChild(dueDate);
+  panel.appendChild(proprity);
+
+  todoDisplayEl.appendChild(todoLiEl);
+  todoLiEl.appendChild(title);
+  todoLiEl.appendChild(input);
+  todoLiEl.appendChild(deleteTaskBtn);
+  todoLiEl.appendChild(panel);
+
+  title.addEventListener('click', () => {
+    title.classList.toggle('active');
+    if (panel.style.display === 'block') {
+      panel.style.display = 'none';
+    } else {
+      panel.style.display = 'block';
+    }
+  });
+
+  deleteTaskBtn.addEventListener('click', (e) => deleteToDo(e));
+};
+
+const createNewProject = (project, index) => {
+  const liEl = document.createElement('li');
+  const lianchorEl = document.createElement('a');
+  lianchorEl.textContent = project.title;
+  lianchorEl.id = `index-${index}`;
+
+  lianchorEl.addEventListener('click', (e) => {
+    document.querySelector('.active-project').classList.remove('active-project');
+    projectIndexTask.value = (e.srcElement.id).split('-')[1];
+    lianchorEl.classList.add('active-project');
+    document.getElementById('createTaskDiv').classList.add('d-block');
+
+    todoDisplayEl.innerHTML = '';
+    for (let j = 0; j < project.items.length; j += 1) {
+      showToDoItemInDom(project.items[j], j);
+    }
+  });
+
+  liEl.appendChild(lianchorEl);
+  return liEl;
+};
+
+const getDataFromLocalStorage = () => {
+  projects = JSON.parse(localStorage.getItem('todolist'));
+  projectsulEl.innerHTML = '';
+  for (let i = 0; i < projects.length; i += 1) {
+    const liEl = createNewProject(projects[i], i);
+    if (i === 0) liEl.getElementsByTagName('a')[0].classList.add('active-project');
+    projectsulEl.appendChild(liEl);
+  }
+};
+
+getDataFromLocalStorage();
+
+const updateLocalStorage = () => {
+  localStorage.setItem('todolist', JSON.stringify(projects));
+};
+
+createProjectBtn.addEventListener('click', () => {
+  const projectName = document.getElementById('projectName').value;
+  const newProject = new Project(projectName);
+  console.log(projects.length);
+  projects.push(newProject);
+  updateLocalStorage();
+  $('#projectModal').modal('hide');
+  createNewProject(newProject, projects.length - 1);
+  getDataFromLocalStorage();
+});
+
+btnCreateTask.addEventListener('click', () => {
+  const taskName = document.getElementById('taskName').value;
+  const description = document.getElementById('description').value;
+  const dueDate = document.getElementById('dueDate').value;
+  const priority = document.getElementById('priority').value;
+
+  const toDo = new Todo(taskName, description, dueDate, priority);
+  const projectIndex = projectIndexTask.value;
+  projects[projectIndex].items.push(toDo);
+  // console.log(toDo[dueDate]);
+  updateLocalStorage();
+  const todoIndex = projects[projectIndex].items.length - 1;
+  showToDoItemInDom(toDo, todoIndex);
+  $('#taskModal').modal('hide');
+});
```

Comment:
- [ ] This module is too big, I suggest that you make more functions, giving them only one responsibility (only one action) instead of doing a lot of things in one function. Then separate it into two parts: `logic.js` and `DOM.js` (example), and then put all of your functions that play with the dom in the second one, and the logic related functions in the first one. This way your code would be a lot more organized.